mix/qml/js/InputValidator.js

```diff
@@ -1,27 +1,20 @@
 Qt.include("QEtherHelper.js")
 
 var nbRegEx = new RegExp('^[0-9]+$');
+var arrayRegEx = new RegExp('\\[[^\\]]*\\]', "g");
```

Comment:
Those regex are used by a few functions here... but I will change this to only instantiate them in init().